zeppelin-web/src/app/interpreter/interpreter.controller.js

```diff
@@ -41,6 +42,21 @@ angular.module('zeppelinWebApp').controller('InterpreterCtrl',
       });
     };
 
+    var checkDownloadingDependencies = function() {
+      var isDownloading = false;
+      for (var setting = 0; setting < $scope.interpreterSettings.length; setting++) {
+        if ($scope.interpreterSettings[setting].status === 'DOWNLOADING_DEPENDENCIES') {
+          isDownloading = true;
+          break;
+        }
+      }
+      if (isDownloading) {
+        $timeout(function() {
+          getInterpreterSettings();
+        }, 2000);
```

Comment:
Yes I thought about it, but I'll take care of this in another PR, where instead of doing a get call, migrate this web-sockets.